cli/cmd/root.go

```diff
@@ -37,4 +40,6 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&output, "output", "o", string(formatters.DefaultOutput), fmt.Sprintf("output format [%s]", outputFormatsString))
 	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "config.yml", "config file will be used by the CLI")
 	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "display debug information")
+
+	rootCmd.PersistentFlags().StringVarP(&overrideEndpoint, "endpoint", "e", "", "endpoint")
```

Comment:
The alias `e` is used for the environment option, we will need to change that.
This is the error that had happened in our pipeline: `panic: unable to redefine 'e' shorthand in "run" flagset: it's already used for "environment" flag`

Reply:
Nice catch. Do we need a short name for this option? We could use just `--endpoint`. Or maybe use something else as `--server-url (-s)`

Reply:
good catch. renamed the flag to overcome conflict